backend/classify.get.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+include_once 'classify.functions.php';
+
+if (!empty($_GET['IP'])) {
```

Comment:
ok, we can leave that part in, might be a good idea to mark it as that then though